src/main/java/com/iexec/sms/secret/Secret.java

```diff
@@ -28,11 +25,11 @@
 import javax.persistence.Id;
 import java.util.Objects;
 
-@Data
-@Getter
-@AllArgsConstructor
 @Entity
-@NoArgsConstructor
+@Builder
```

Comment:
Why do you add this annotation if you don't use it ?

Reply:
I planned to use it... But finally removed its usages. So annotation can be removed. Please see https://github.com/iExecBlockchainComputing/iexec-sms/pull/134/commits/0387d46e55e63b64dfef293b2f07d9f8cb715961.